keyboards/helix/rev2/keymaps/froggy_106/keymap.c

```diff
@@ -0,0 +1,901 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
+#include "keymap_jp.h"
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+#ifdef AUDIO_ENABLE
+  #include "audio.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
```

Comment:
```suggestion
```

Reply:
Include directives for bootloader.h, lufa.h, split_util.h, audio.h are removed, they are not needed.